src/layer/tile/GridLayer.js

```diff
@@ -413,15 +413,19 @@ L.GridLayer = L.Layer.extend({
 		if (!map) { return; }
 
 		if (center === undefined) { center = map.getCenter(); }
-		if (zoom === undefined) { zoom = Math.round(map.getZoom()); }
+		if (zoom === undefined) { zoom = map.getZoom(); }
 
 		if (zoom > this.options.maxZoom ||
 			zoom < this.options.minZoom) { return; }
```

Comment:
yeah, I mean, e.g. with `zoom = 14.2` and `maxZoom = 14`, now it'll return early, whereas before the PR it wouldn't because it compares the rounded value.